_config.yml

```diff
@@ -119,14 +119,10 @@ exclude:
 plugins:
```

Comment:
In the "exclude:" block above, add the folder containing the feed R scripts

Reply:
Done in https://github.com/hpc-carpentry/hpc-carpentry.github.io/pull/25/commits/59c6fd76f582e98d9ddf1d91020559401b972b56